### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by 0rl1s**
Finished feedback feature.

### CODE REVIEW

File: Pages/FeedbackPage/Feedback.cshtml [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+
+            <!--<div class="row">
+        <div class="col-md-4">
```
Comment (ArmantasPik) [inline Pages/FeedbackPage/Feedback.cshtml:91]: Is all of this commented section needed?

File: Pages/FeedbackPage/FeedbackLogged.cshtml.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        private readonly FoodKept.Data.ShopContext _context;
+        private readonly UserManager<ApplicationUser> _userManager;
+        public FeedbackLoggedModel(FoodKept.Data.ShopContext context, UserManager<ApplicationUser> userManager)
+        {
+            _context = context;
+            _userManager = userManager;
+        }
+        [BindProperty]
+        public Feedback Feedback { get; set; }
+        public ApplicationUser ApplicationUser { get; set; }
```
Comment (ArmantasPik) [inline Pages/FeedbackPage/FeedbackLogged.cshtml.cs:24]: Is this needed? (ApplicationUser)

File: Pages/Shared/_Layout.cshtml [hunk 3]
```diff
@@ -26,6 +26,7 @@
                 <div class="navbar-collapse collapse d-sm-inline-flex justify-content-between">
                     @if (signInManager.IsSignedIn(User))
                     {
+
```
Comment (ArmantasPik) [inline Pages/Shared/_Layout.cshtml:29]: Space not needed

File: Pages/FeedbackPage/Feedback.cshtml (hunk 1 above)
Comment (0rl1s) [reply Pages/FeedbackPage/Feedback.cshtml:91]: Not anymore. Forgot to delete it but will do.

**Review by ArmantasPik — APPROVED** [review]

**Comment by medikornov** [comment]
nothing to add, Armantas mentioned everything, GJ

File: Pages/FeedbackPage/FeedbackLogged.cshtml.cs (hunk 2 above)
Comment (0rl1s) [reply Pages/FeedbackPage/FeedbackLogged.cshtml.cs:24]: At this moment not really but in the future it could be.

**Pull request merged**